plugins/woocommerce-admin/client/launch-your-store/hub/sidebar/xstate.tsx

```diff
@@ -255,38 +294,47 @@ export const sidebarMachine = setup( {
 						assign( { launchStoreError: undefined } ), // clear the errors if any from previously
 						'recordStoreLaunchAttempt',
 					],
-					invoke: {
-						src: 'updateLaunchStoreOptions',
-						onDone: {
-							target: '#storeLaunchSuccessful',
-							actions: [
-								{
-									type: 'recordStoreLaunchResults',
-									params: { success: true },
-								},
-							],
-						},
-						onError: {
-							actions: [
-								assign( {
-									launchStoreError: ( { event } ) => {
-										return {
-											message: JSON.stringify(
-												event.error
-											), // for some reason event.error is an empty object, worth investigating if we decide to use the error message somewhere
-										};
+					invoke: [
+						{
+							src: 'updateLaunchStoreOptions',
+							onDone: {
+								target: '#storeLaunchSuccessful',
+								actions: [
+									{
+										type: 'recordStoreLaunchResults',
+										params: { success: true },
 									},
-								} ),
-								{
-									type: 'recordStoreLaunchResults',
-									params: {
-										success: false,
+								],
+							},
+							onError: {
+								actions: [
+									assign( {
+										launchStoreError: ( { event } ) => {
+											return {
+												message: JSON.stringify(
+													event.error
+												), // for some reason event.error is an empty object, worth investigating if we decide to use the error message somewhere
+											};
+										},
+									} ),
+									{
+										type: 'recordStoreLaunchResults',
+										params: {
+											success: false,
+										},
 									},
-								},
-							],
-							target: '#launchYourStoreHub',
+								],
+								target: '#launchYourStoreHub',
+							},
 						},
-					},
+						{
+							src: 'deleteTestOrders',
+							input: ( { event } ) => ( {
+								// @ts-expect-error - how do I type this?
+								removeTestOrders: event.removeTestOrders,
```

Comment:
Hey @rjchow ✋ 

I wasn't able to find a way to type this. Could you help me on this please 🙏 ? I've tried a few different types and they all threw me errors :(